fizzbuzz/FizzBuzz.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+class FizzBuzz
+{
+    private $rules = [];
+
+    public function __construct()
+    {
+        $this->rules[] = new MultipleOfThree();
+        $this->rules[] = new MultipleOfFive();
+    }
+
+    public function check($number)
+    {
+        $output = '';
+        foreach ($this->rules as $rule) {
+            $output .= $rule->output($number);
+        }
+
+        if (strlen($output)) {
+            return $output . PHP_EOL;
+        }
+        return $number . PHP_EOL;
+    }
+}
+
+abstract class Multiple
+{
+    abstract public function output($number);
+}
+
+class MultipleOfThree extends Multiple
+{
+    public function output($number)
+    {
+        if ($number % 3 === 0) {
+            return 'Fizz';
+        }
+    }
+}
+class MultipleOfFive extends Multiple
```

Comment:
There is duplication between the classes `MultipleOfThree` and `MultipleOfFive`, since they differ only on the number and word used. Therefore, try to unify them t a single class and configure it through its constructor, passing in these parameters